dmapiclient/data.py

```diff
@@ -330,6 +330,11 @@ def export_users(self, framework_slug):
             "/users/export/{}".format(framework_slug)
         )
 
+    def email_address_has_valid_buyer_domain(self, email_address):
```

Comment:
I know what you mean, but I think there is some use in doing something to distinguish API client methods that are returning a boolean rather than a JSON response (there is another one coming soon to check if a supplier is eligible to apply to a brief once this gets merged: https://github.com/alphagov/digitalmarketplace-api/pull/359).  Whether this is the best naming convention I'm not sure, but it's something at least...